<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    
    background-color:#cf4f42;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: rgb(211, 211, 211)
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
    background-color:#cf4f42;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>30</number>
    </property>
    <property name="leftMargin">
     <number>60</number>
    </property>
    <property name="topMargin">
     <number>100</number>
    </property>
    <property name="rightMargin">
     <number>60</number>
    </property>
    <item>
     <widget class="QLabel" name="imageLabel">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="loginField">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {

    border: 10px;
    border-color: red;
    border-radius: 20px;
    
    background: white;
    padding: 23 px;
    padding-left:23px;
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordField">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px rgb(208, 208, 208);
    border-radius: 20px;

    background: white;
    selection-background-color: darkgray;
    padding: 23px;
    padding-left:23px;
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="enterButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 3px;
    border-radius: 20px;
    background-color:#cf4f42;
    color:white;
    border-style:outset;
border-color:white;
    padding: 20px;
}

QPushButton:focus {
	outline: none;
}

QPushButton:pressed{
    background-color: #792d26;
}

</string>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{

    color:white;
}</string>
      </property>
      <property name="text">
       <string>All rights reserved.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
